Library/ViewModels/ActivitiesViewModel.swift

```diff
@@ -169,6 +169,7 @@ public final class ActivitiesViewModel: ActivitiesViewModelType, ActitiviesViewM
     .ignoreValues()
 
     self.updateUserInEnvironment = self.clearBadgeValue
+      .filter { _ in AppEnvironment.current.currentUser != nil }
```

Comment:
Do we need to add a test for this in `ActivitiesViewModelTests`

Reply:
Woops, yes it should thanks.